source/queue.ts

```diff
@@ -2,6 +2,7 @@ export type RunFunction = () => Promise<unknown>;
 
 export interface Queue<Options> {
```

Comment:
What do you think of:

```suggestion
export interface Queue<Element = RunFunction, Options> {
```

?

Reply:
I _think_ that the default type throws off callers which just provide one param for `Options`.